agent/api/task/task.go

```diff
@@ -720,8 +720,6 @@ func (task *Task) updateTaskKnownStatus() (newStatus apitaskstatus.TaskStatus) {
 			containerEarliestKnownStatus.String(), task)
 		return apitaskstatus.TaskStatusNone
 	}
-	seelog.Debugf("Container with earliest known container is [%s] for task: %s",
```

Comment:
why is this deleted?

Reply:
@petderek: lol why did we do this again?

Reply:
Its duplicated elsewhere.

Reply:
er wait, no its not. I'll re-add this